marklogic/rows.py

```diff
@@ -40,4 +41,113 @@ def graphql(self, graphql_query, return_response=False, *args, **kwargs):
             response.json()
             if response.status_code == 200 and not return_response
             else response
-        )
\ No newline at end of file
+        )
+
+    def query(self, dsl=None, plan=None, sql=None, sparql=None, format="json", return_response=False, *args, **kwargs):
```

Comment:
One small tweak here that I should have mentioned in the previous PR - define types for each of these parameters - e.g. 

    self, dsl: str = None, plan: dict = None, ...